### PR TITLE
sql: add some new tidb specific system variables

### PR DESCRIPTION
**Pull request by XuHuaiyu**
Add description for some new tidb specific system variables.

### CODE REVIEW

**Comment by XuHuaiyu** [comment]
PTAL @coocood

File: sql/tidb-specific.md [hunk 1]
```diff
@@ -294,6 +318,37 @@ TiDB 在 MySQL 的基础上，定义了一些专用的系统变量和语法用
...
+
+这个变量用来设置是否开启 TABLE PARTITION 特性。
+
+## tidb_backoff_lock_fast
+
+作用域：SESSION | GLOBAL
+
+默认值：100
+
+这个变量用来设置读请求遇到锁的 backoff 时间
```
Comment (lilin90) [inline sql/tidb-specific.md]: Please also add a period in the end of the sentence, to keep it consistent with others.

File: sql/tidb-specific.md [hunk 2]
```diff
@@ -294,6 +318,37 @@ TiDB 在 MySQL 的基础上，定义了一些专用的系统变量和语法用
...
+
+这个变量用来设置 DDL 操作 re-organize 阶段的并发度。
+
+## tidb_ddl_reorg_priority
+
+作用域：SESSION | GLOBAL
+
+默认值：PRIORITY_NORMAL
+
+这个变量用来设置 ADD INDEX 操作 re-organize 阶段的执行优先级, 可设置为 PRIORITY_LOW/PRIORITY_NORMAL/PRIORITY_HIGH。
```
Comment (lilin90) [inline sql/tidb-specific.md]: - Add the inline code format for `ADD INDEX`.
- Modify the comma after "优先级" from the English version to the Chinese version, and delete the extra space after comma.

**Comment by coocood** [comment]
LGTM

**Pull request merged**